com.vrcfury.vrcfury/Editor/VF/Feature/ToggleBuilder.cs

```diff
@@ -207,15 +245,22 @@ VFABool physBoneResetter
         }
 
         if (model.enableDriveGlobalParam && !string.IsNullOrWhiteSpace(model.driveGlobalParam)) {

```

Comment:
These null / empty checks aren't needed because now it's handled in the new method